docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md

```diff
@@ -0,0 +1,474 @@
+                            "input": "vendor/spryker/spryker/Bundles",
+                            "output": "/assets/"
+                        },
+                        {
+                            "glob": "*/Presentation/Components/assets/**/*",
+                            "input": "src/Pyz/Zed",
+                            "output": "/assets/"
+                        }
+                    ],
+                    "styles": [
+                        "vendor/spryker/spryker/Bundles/ZedUi/src/Spryker/Zed/ZedUi/Presentation/Components/styles.less",
+                        "src/Pyz/Zed/ZedUi/Presentation/Components/styles.less"
+                    ],
+                    "scripts": []
+                },
+                "configurations": {
+                    "development": {
+                        "buildOptimizer": false,
+                        "optimization": false,
+                        "vendorChunk": true,
+                        "extractLicenses": false,
+                        "sourceMap": true,
+                        "namedChunks": true
+                    },
+                    "production": {
+                        "fileReplacements": [
+                            {
+                                "replace": "src/Pyz/Zed/ZedUi/Presentation/Components/environments/environment.ts",
+                                "with": "src/Pyz/Zed/ZedUi/Presentation/Components/environments/environment.prod.ts"
+                            }
+                        ],
+                        "optimization": true,
+                        "outputHashing": "none",
+                        "sourceMap": false,
+                        "namedChunks": false,
+                        "extractLicenses": true,
+                        "vendorChunk": true,
+                        "buildOptimizer": true,
+                        "budgets": [
+                            {
+                                "type": "bundle",
+                                "maximumWarning": "2mb",
+                                "maximumError": "5mb"
+                            }
+                        ]
+                    }
+                },
+                "defaultConfiguration": "development"
+            },
+            "lint": {
+                "executor": "@nx/eslint:lint",
+                "options": {
+                    "eslintConfig": ".eslintrc.mp.json",
+                    "lintFilePatterns": [
+                        "src/Pyz/Zed/*/Presentation/Components/**/*.ts",
+                        "src/Pyz/Zed/*/Presentation/Components/**/*.html"
+                    ],
+                    "noEslintrc": true
+                }
+            },
+            "test": {
+                "executor": "@nx/jest:jest",
+                "options": {
+                    "jestConfig": "frontend/merchant-portal/jest.config.ts"
+                }
+            }
+        }
+    }
+   ```
+
+   6. In `deploy.yml` update npm to `10` version.
```

Comment:
We also have to update requirement section of setup page for MP: https://docs.spryker.com/docs/dg/dev/frontend-development/202311.0/marketplace/set-up-the-merchant-portal.html#requirements

Also probably update https://docs.spryker.com/docs/dg/dev/system-requirements/202311.0/system-requirements.html#marketplace-system-requirements

Reply:
Revert npm